packages/nextjs/src/server/clerkClient.ts

```diff
@@ -27,6 +34,63 @@ const clerkClient = createClerkClient({
     disabled: TELEMETRY_DISABLED,
     debug: TELEMETRY_DEBUG,
   },
+};
+
+export const clerkClientStorage = new AsyncLocalStorage<Partial<AuthenticateRequestOptions>>();
+
+const createClerkClientWithOptions: typeof createClerkClient = options =>
+  createClerkClient({ ...clerkClientDefaultOptions, ...options });
+
+const clerkClientSingleton = createClerkClient(clerkClientDefaultOptions);
+
+/**
+ * @deprecated
+ * This singleton is deprecated and will be removed in a future release. Please use `clerkClient()` as a function instead.
+ */
+const clerkClientSingletonProxy = new Proxy(clerkClientSingleton, {
+  get(target, prop, receiver) {
+    deprecated('clerkClient object', 'Use `clerkClient()` as a function instead.');
+
+    return Reflect.get(target, prop, receiver);
+  },
```

Comment:
How does this work with nested properties? E.g. if I access `clerkClient.users.getUser()` will this still log the deprecation warning?